### PR TITLE
docs: add guide on granting gitlab admin role using cli

### PR DESCRIPTION
**Pull request by wesjdj**
Add a docs section on how to grant a GitLab user the Gitlab admin role using the GitLab CLI.

### CODE REVIEW

File: docs/how-to-guides/admin/deploying-renku.rst [hunk 1]
```diff
@@ -259,6 +259,17 @@ If the RenkuLab deployment is relying on an existing GitLab instance, this insta
 #. Set ``Alias`` to <renku-domain>, ``Authorization URL`` to \https://<gitlab-domain>/oauth/authorize, ``Token URL`` to \https://<gitlab-domain>/oauth/token and ``Client ID`` and ``Client Secret`` to the values used in step 1.
 #. Click ``Save``. The new Identity Provider should appear and any user from the stand-alone GitLab instance should be able to login.
 
+Grant a user the GitLab admin role using the GitLab CLI
```
Comment (rokroskar) [inline docs/how-to-guides/admin/deploying-renku.rst:262]: I think this should probably either replace or merge with the section above that is now called "GitLab deployed as part of renku"

**Review by rokroskar — CHANGES_REQUESTED** [review]
Thanks, this is much simpler than the previous instructions. Please either replace the previous instructions or merge them together under a joint section.

**Comment by rokroskar** [comment]
@wesjdj what's the status here?

**Comment by wesjdj** [comment]
I'm going to close this PR as I've included this doc change in my wider GitLab docs rework PR #2741

**Pull request closed without merging**